constants/constants.go

```diff
@@ -2,3 +2,77 @@ package constants
 
 // CustomDateFormat is a date format used in the application to parse date string/object into a usable format
 const CustomDateFormat = "2006-01-02"
+
+// constants defined to use in ActionItemTypeMap
+const (
+	Retrospective         = "Retrospective"
+	RetrospectiveFeedback = "RetrospectiveFeedback"
+	SprintMember          = "SprintMember"
+	SprintMemberTask      = "SprintMemberTask"
+	Sprint                = "Sprint"
+	SprintTask            = "SprintTask"
+)
+
+// ActionItemTypeMap is types of ActionItem of Trail model used in adding trails.
+var ActionItemTypeMap = map[string]string{
+	Retrospective:         "Retrospective",
+	RetrospectiveFeedback: "Retrospective Feedback",
+	SprintMember:          "Sprint Member",
+	SprintMemberTask:      "Sprint Member Task",
+	Sprint:                "Sprint",
+	SprintTask:            "Sprint Task",
+}
+
+// constants defined to use in ActionTypeMap
+const (
+	CreatedRetrospective    = "CreatedRetrospective"
+	AddedGoal               = "AddedGoal"
+	UpdatedGoal             = "UpdatedGoal"
+	ResolvedGoal            = "ResolvedGoal"
+	UnresolvedGoal          = "UnresolvedGoal"
+	AddedHighlight          = "AddedHighlight"
+	UpdatedHighlight        = "UpdatedHighlight"
+	AddedSprintMember       = "AddedSprintMember"
+	UpdatedSprintMember     = "UpdatedSprintMember"
+	RemovedSprintMember     = "RemovedSprintMember"
+	AddedNote               = "AddedNote"
+	UpdatedNote             = "UpdatedNote"
+	AddedSprintMemberTask   = "AddedSprintMemberTask"
+	UpdatedSprintMemberTask = "UpdatedSprintMemberTask"
+	CreatedSprint           = "CreatedSprint"
+	DeletedSprint           = "DeletedSprint"
+	UpdatedSprint           = "UpdatedSprint"
+	ActivatedSprint         = "ActivatedSprint"
+	FreezeSprint            = "FreezeSprint"
+	TriggeredSprintRefresh  = "TriggeredSprintRefresh"
+	UpdatedSprintTask       = "UpdatedSprintTask"
+	MarkDoneSprintTask      = "MarkDoneSprintTask"
+	MarkUndoneSprintTask    = "MarkUndoneSprintTask"
+)
+
+// ActionTypeMap is types of Action of Trail model used in adding trails.
+var ActionTypeMap = map[string]string{
```

Comment:
Please check if all the actions are present and if the text is making sense if used in a statement, avoid un-necessary capital words. Let me know if an action is not clear before making a statement for it.